documentation/modules/cruise-control/ref-cruise-control-configuration.adoc

```diff
@@ -0,0 +1,92 @@
+// Module included in the following assemblies:
+//
+// assembly-cruise-control-concepts.adoc
+
+[id='ref-cruise-control-configuration-{context}']
+= Cruise Control configuration
+
+The `config` property in `Kafka.spec.cruiseControl` contains configuration options as keys with values as one of the following JSON types:
+
+* String
+* Number
+* Boolean
```

Comment:
This applies more generally to the rest of the docs too, but here would be a good place for a `NOTE` to say that if your String happens to look like YAML or JSON it will need to be explicitly quoted.